.github/workflows/test.yml

```
@@ -0,0 +1,66 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        os: [
+          "ubuntu-18.04",
+          "macOS-10.14",
+        ]
+        python-version: [
+          "pypy3",
+          "3.7",
+          "3.5",
+          "3.6",
+        ]
+        include:
+        - python-version: "3.5"
+          env: PYTHONOPTIMIZE=2
+        - python-version: "3.6"
+          env: PYTHONOPTIMIZE=1
+    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.os }} Python ${{ matrix.python-version }}
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Linux dependencies
+      if: matrix.os == 'ubuntu-18.04'
```

Untested, but what about this?
```suggestion
      if: matrix.os =! ^ubuntu
```

---

Thanks, updated with `startsWith`.